packages/sdk/wallets/wallet-base/src/signing-utils.test.ts

```diff
@@ -270,7 +270,6 @@ function ckToViem(tx: CeloTx & { v?: number }) {
 }
 
 describe('recoverTransaction', () => {
-  const ACCOUNT_ADDRESS1 = privateKeyToAddress(PRIVATE_KEY1)
```

Comment:
it looks like removing this broke a test.  seems like solution will be to rename the const here and on line 290/291 expect(signer).toEqual(ACCOUNT_ADDRESS1)